apps/ejabberd/src/foreign_event/mod_foreign_http.erl

```diff
@@ -28,9 +28,11 @@
 %%--------------------------------------------------------------------
 
 make_request(Request, OnResponse) ->
-    {Host, Path, Method, Headers, Body} = parse(Request),
-    Response = do_make_request(Host, Path, Method, Headers, Body),
-    respond(encode(Response), OnResponse),
+    spawn(fun() ->
```

Comment:
I would like to receive IQ result after I've made a request.
Or an IQ error if the server cannot contact http server or any other problem.